### DISCUSSION TITLE
Changement de nom de fournisseur de variables

### DISCUSSION THREAD

**Discussion by JYK-WIT** [witsa/synapps#617] [Q&A]
Bonjour,
est il possible, lors d'un changement de clé de fournisseur de variable, que les liens utilisant cette clé soient aussi changé ?
Merci.

**Comment by manufitoussiwit** [comment]
Bonjour,
Pour l'instant, dans le designer, lorsqu'on change la clé d'un acteur, si des liaisons pointes dessus, elles ne sont pas mise à jour automatiquement avec la nouvelle clé. C'est pareil avec les clés de librairies. 
C'est une fonctionnalité à venir.

J'ai une astuce en attendant dans d'autres cas : 
Sur une propriété qui attend la clé d'un acteur, comme par exemple la clé du fournisseur parent sur un fournisseur de variable relative, je place une liaison vers la clé de l'acteur en question. Comme ça si la clé change, ça suit. 
Parfois je le fais aussi pour les clés d'acteurs que j'utilise dans un script. Je crée des additionnelles qui sont liées aux clés des acteurs dont j'ai besoin dans un script.
